great_expectations/rule_based_profiler/util.py

```diff
@@ -17,6 +17,25 @@
 NP_EPSILON: Union[Number, np.float64] = np.finfo(float).eps
 
 
+def validate_builder_config(builder_config: dict):
+    if not (
+        isinstance(builder_config, dict)
+        and len(
+            (
+                set(builder_config.keys())
+                & {
+                    "class_name",
+                    "module_name",
+                }
+            )
+        )
+        == 2
+    ):
+        raise ge_exceptions.ProfilerConfigurationError(
+            'Both "class_name" and "module_name" must be specified.'
+        )
```

Comment:
Please add a docstring here and in all other public functions.

Using the `Raises:` section of the Google style guide please!

Reply:
Also, can we simplify this check? We don't need to do any bitwise operators or set conversion in my opinion.

```python
if not isinstance(builder_config, dict) or any(attr in builder_config for attr in ("class_name", "module_name"):
    raise
```

Reply:
Per discussion with @donaldheppner in my own PR, I agree with his approach of making methods like return boolean values. Validation should be true/false and an exception should be raised at the same level as function invocation.

Reply:
@cdkini I will add the docstring (in the next commit).  Please note that this is not a boolean check, but must raise an exception (as in -- "cannot continue" -- I will document this.  Also, this ins not a bitwise operation (it is the recommended `set` shorthand for `intersection`).